downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/HumanPathwaysWithDiagrams.java

```diff
@@ -38,19 +47,57 @@ public static void execute(MySQLAdaptor dba, String releaseNumber) throws Except
             boolean isDisease = false;
 
             // If returned value from 'disease' query is not null, its considered to be a disease-containing Pathway
-            if (humanPathway.getAttributeValue(ReactomeJavaConstants.disease) != null) {
+            if (isDiease(humanPathway)) {
                 isDisease = true;
             }
+;
             Collection<GKInstance> representedPathwayInstances = humanPathway.getReferers(ReactomeJavaConstants.representedPathway);
             // If returned value from 'representedPathway' referral is not null, it is considered to have a pathway diagram
+
             if (representedPathwayInstances != null) {
-                // Generate line in file and append to that file
-                String line = humanPathway.getDBID() + "\t" + humanPathway.getDisplayName() + "\t" + isDisease + "\n";
-                Files.write(Paths.get(humanPathwaysWithDiagramsFilename), line.getBytes(), StandardOpenOption.APPEND);
+                if (!onlyGreenBoxDiagrams(representedPathwayInstances)) {
```

Comment:
Could this condition be joined with the outer one with `&&` since there's no else branch?

Reply:
Done